Mathlib/RingTheory/Ideal/Quotient/Index.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2024 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Algebra.GeomSum
+import Mathlib.Data.Finsupp.Fintype
+import Mathlib.GroupTheory.Index
+import Mathlib.LinearAlgebra.DirectSum.Finsupp
+import Mathlib.LinearAlgebra.TensorProduct.Quotient
+import Mathlib.LinearAlgebra.TensorProduct.RightExactness
+import Mathlib.RingTheory.Finiteness.Cardinality
+import Mathlib.RingTheory.Finiteness.TensorProduct
+import Mathlib.RingTheory.Ideal.Quotient.Operations
```

Comment:
Do we really need all of those? What does `#min_imports` say?

Reply:
Yes. I believe CI would complain otherwise.

Reply:
Oh wow, we really do need them all